tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -278,7 +277,7 @@ func parseManifestListForSnapshotIDs(output string) []string {
 // waitUntilServerStarted returns error if the Kopia API server fails to start before timeout
 func (ks *KopiaSnapshotter) waitUntilServerStarted(ctx context.Context, addr string, serverStatusArgs ...string) error {
 	statusArgs := append([]string{"server", "status", "--address", addr}, serverStatusArgs...)
-	if err := retry.PeriodicallyNoValue(ctx, retryInterval, retryCount, waitingForServerString, func() error {
+	if err := retry.PeriodicallyNoValue(ctx, retryInterval, retryCount, waitingForServerString, func() error { //nolint:wsl
```

Comment:
Can this linter warning be addressed instead of silencing it?

Reply:
Agreed, will fix them in the PR for robustness-suite, this needs to be rebased and then cleaned up.